src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -436,28 +468,33 @@ void Instance::OnFinished(Status status, CharSpan debugText, ThreadScanResponseI
     TLV::TLVType listContainerType;
     ThreadScanResponse scanResponse;
     size_t networksEncoded = 0;
+    uint8_t extendedAddressBuffer[8];
```

Comment:
That `8` should be a constant defined somewhere.  I would be very surprised if we do not have a constant for this already.